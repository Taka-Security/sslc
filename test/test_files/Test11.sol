contract A {
  struct StructB_return_early {
    bytes12 var1;
    uint160 var2;
    uint88 var1;
    bytes24 var4;
    bytes10 var5;
    bytes11 var6;
  }
}