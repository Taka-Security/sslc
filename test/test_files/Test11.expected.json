[
    {
        "file": "Test11.sol",
        "contract": "A",
        "struct": "StructB_return_early",
        "slotsSaved": 1,
        "timedout": false
    }
]